docs/source/API/containers/ScatterView.rst

```diff
@@ -0,0 +1,126 @@
+``ScatterView``
+===============
+
+.. role:: cppkokkos(code)
+	:language: cppkokkos
+
+Header File: ``<Kokkos_ScatterView.hpp>``
+
+.. _parallelReduce: ../core/parallel-dispatch/parallel_reduce.html
+
```

Comment:
Why is `parallel_reduce` here?  It's not directly related to ScatterView.

Reply:
it is just a link